cloudigrade/api/error_codes.py

```diff
@@ -168,8 +168,9 @@ def notify(self, account_number, application_id, error_message=None):
     "CG3000",
     _("Failed to enable Cloud Account %(cloud_account_id)s: " "because %(exception)s"),
```

Comment:
I know it wasn't part of this commit (probably an old black reformatting artifact), but there are several split strings like this in the nearby context that we could fix while we're here.
```suggestion
    _("Failed to enable Cloud Account %(cloud_account_id)s: because %(exception)s"),
```